src/utils/nssp.py

```diff
@@ -0,0 +1,92 @@
+"""
+Utils for pulling data from NSSP ETL.
+"""
+
+from dataclasses import dataclass
+from datetime import date
+
+import polars as pl
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import ContainerClient
+
+from src.utils.az import AzureStorage, scan_blob_parquet
+
+
+@dataclass
+class NSSP:
```

Comment:
I really like this idea! After a lot of pain and annoyance around "is it 'disease' or 'pathogen'?" and "is it 'state' or 'geo_value'", this makes so much sense.